node-admin/src/main/resources/configdefinitions/config-server.def

```diff
@@ -18,3 +18,5 @@ keyStoreConfig.password      string                      default=""
 trustStoreConfig.path        string                      default=""
 trustStoreConfig.type        enum { JKS, PEM, PKCS12 }   default=JKS
 trustStoreConfig.password    string                      default=""
+
+jvmargs                      string                      default=""
```

Comment:
Is this option for config server only or all nodes in zone?
If its the first, then we don't need to add this here, adding it to `Envrionment.Builder` in `/vespa/hosted` should be enough

If its the later: this config is for all node-admins, and contains only config related to who the config servers for this node are and how to talk to them. `jvmargs` is neither of those things, so we would need either a new general node-admin config and move some of the environment variables there, or pass this as environment variable.

Maybe consider adding some ContainerEnvironmentSettings "resolver" like `InetAddressResolver` or `PathResolver` we already have in `Enviroment`. That resolver would implement `String createContainerEnvironmentSettings(Environment, ContainerNodeSpec)`, then we could easily add new options in hosted code without having to extend `Environment` every time.

Reply:
This is for config server only, so yes, it should be enough to add it to Environment. Adding resolver sounds reasonable, I'll look into it.